sql/custom/world/2016_03_19_00_qc_world.sql

```diff
@@ -0,0 +1,3 @@
+-- Blinding Light
```

Comment:
Since the intention is to include it in the TC repo, the sql should be added to /sql/updates/world